File: doFib/fib.go
package main

import (
	"fmt"
	"time"
)
	
func fib(n int) int {
	if n == 1 || n == 2{
		return 1
	} else {
		return fib(n - 1) + fib(n - 2)
	}
}

func main(){
	start := time.Now()
	result := fib(40)
	end := time.Now()
	time_diff := end.Sub(start)
	fmt.Println("斐波那契数列第40项的值为：",result,"用时:",time_diff)
}

File: doFib/fib.py
import time

def fib(n):
	if n in [1,2]:
		return 1
	return fib(n-1)+fib(n-2)	

start = time.time()
result = fib(40)
end = time.time()
print(f"斐波那契数列的第40项是:{result},耗时:{end-start}秒")	